app/layout.tsx

```diff
@@ -13,8 +13,8 @@ const geistMono = Geist_Mono({
 });
 
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "PawRide - Uber for Dogs",
```

Comment:
omg lowercase in the title? we ain't basic here, capitalize 'for' in 'Uber for Dogs' 🤓